README.md

```diff
@@ -1,139 +1,207 @@
-```shell
-go generate
-```
-- This command generates documentation based on the templates in the templates' directory and the MarkdownDescription available in the resources schema..
-
-## Terraform Manifest file(terraform-registry-manifest.json)
-
-- **version** is the numeric version of the manifest format, not the version of our provider.
-- **protocol_versions** is the Terraform protocol version. This is set to 6.0 because we are using Terraform Plugin Framework to develop our provider.
-
-## Debugging in IntelliJ IDEA using Delve
-- There are four IntelliJ IDEA run configurations in `.run` folder. Open IntelliJ run configuration menu to see them.
-- Steps to debug Terraform Provider
-  - brew install delve
-  - Run "Axual build to _bin" to generate executable in `/go/bin` and name it terraform-provider-axual
-  - Run "Axual Delve binary" (Run icon) to start Delve's debugging session on this binary. It looks for terraform-provider-axual executable in `/go/bin`.
-  - Run "Axual Go Remote" (Debug icon) to connect IntelliJ to the Delve's debugging session. On the console you will see a string: TF_REATTACH_PROVIDERS='{"axual...
-  - Copy this string and use it as environment variable: export TF_REATTACH_PROVIDERS='{"axual...
-    - For some reason, it only works in a separate terminal session on MacOS. Also run terraform commands from the same iTerm terminal session.
-  - Set a breakpoint and run a command to run the provider like: tf plan or tf apply. It will stop at the breakpoint
-  - When done with debugging, remove the env variable: unset TF_REATTACH_PROVIDERS
-  - To do the same without generating the binary: Run "Axual Delve project" (Run icon)
-
-
-## Debugging webclient
-- The module axual-webclient-exec is used to test and debug axual-webclient module without any Terraform functionality.
-
-## Logging
-- Logging in axual-webclient:
-  - log.Println using the module "log"
-  - For example:
-    - log.Println("strings.NewReader(string(marshal))", strings.NewReader(string(marshal)))
-- Logging in internal folder:
-  - tflog.Debug(or Info,Trace, etc)
-  - Then run TF_LOG=DEBUG(Or INFO,TRACE,ERROR) terraform plan
-  - For example:
-    - marshal, err := json.Marshal(ApplicationRequest)
-    - tflog.Error(ctx, "MARSHAL", map[string]interface{}{
-      "MARSHAL": string(marshal),
-      })
-    - Run: TF_LOG=ERROR terraform apply -auto-approve
-
-## Acceptance tests
-
-- Please read how to run tests from `DEVELOPERS.md`
\ No newline at end of file
+## Contributing
+
+Contributions are welcome! If you'd like to contribute to the development of this provider, please see [DEVELOPERS.md](DEVELOPERS.md) for information on:
+- Setting up a local development environment
+- Building the provider from source
+- Running tests
+- Debugging
+
+## Support
+
+For issues, questions, or feature requests, please open an issue in this repository.
+
+## License
+
+See [LICENSE](LICENSE) for details.
```

Comment:
This license link is broken.

<img width="1662" height="489" alt="image" src="https://github.com/user-attachments/assets/65853834-512c-47f2-a9c8-d7e05d0ebebd" />